geomesa-core/src/test/scala/org/locationtech/geomesa/core/data/FeatureWritersTest.scala

```diff
@@ -103,7 +103,6 @@ class FeatureWritersTest extends Specification {
 
         /* write the feature to the store */
         fs.addFeatures(featureCollection)
-        fs.flush()
```

Comment:
The flush here was actually some implicit specs2 conversion to ConsoleOutput, which has a flush method...